sale_order_import/wizard/import_sale.py

```diff
@@ -124,8 +124,17 @@ def _get_order_value_dict(self, row, error_line_vals, partner_id,
         else:
             self._get_team_dict(team_value, team_dict, error_line_vals)
 
+        carrier_value = row[carrier_id].strip()
+        if not carrier_value:
+            error_line_vals['error_name'] = error_line_vals['error_name']\
+                + _('Carrier is empty!') + '\n'
```

Comment:
For the all error messages, please apply the following format:
'Column "XXX" cannot be empty.' / 'Column "XXX" cannot be less than 0.'